### ISSUE TITLE
introduzione concetto di Tag

### ISSUE THREAD

**Issue by masierim** [bacca87/great2#162]
propongo di inserire "qualcosa", che serva a raggruppare vari inserimenti di ore (ovvero giornate) con l'obiettivo di avere una statistica delle ore lavorate per progetto.

un progetto può essere composto da varie attività (software interno, collaudo, riunione,avviamento, ecc... ) per ora Great è incentrato alle trasferte ma si potrebbe allargarlo anche alle attività interne.

in questo modo il software potrà fornirci una statistica delle ore/giorni lavorati per questo (fantomatico) Gruppo

la mia idea rispecchia i Tag che si danno alle immagini nei software di foto, come per esempio su Picasa

e non limiterei ad 1 il numero di Tag che puoi dare all'inserimento.

es. se io inserisco questi Tag :  CC1449, F2ZZ0234, SWPLIN, COPLIN.
 assocerò all'inserimento o a più inserimenti : { CC1449, F2ZZ0234, SWPLIN}
ed ad altri{ CC1449, F2ZZ0234, COPLIN}

in questo modo avrò una statistica delle ore lavorate per SWPLIN, quella per la commessa CC1449 e quella, più specifica, per la macchina F2ZZ0234.

sarà poi a diserzione del utilizzatore il metodo di utilizzo delle Tag (per come più gli piace).

se messo in lista, può essere utile anche per risalire ad un giorno specifico dell'anno.

non limiterei la questione solo ai giorni lavorati, ma a tutti i tipi.

per info, dubbi, perplessità o se( più probabile...) non si capisce un cazzo. ne parliamo a voce....

**Comment by corradinia91** [comment]
Dici che si possono riutilizzare le note del TS per questo scopo?

**Comment by bacca87** [comment]
Si l'idea era quella, di inserire tipo degli hashtag #

**Comment by corradinia91** [comment]
Ma in un campo nuovo?

Ottieni Outlook<https://aka.ms/qtex0l> per iOS
________________________________

**Comment by bacca87** [comment]
No, i tag andrebbero ricercati all'interno del testo delle note.
La gestione però va pensata bene, perche se no le query diventano pesanti.

**Comment by corradinia91** [comment]
Assegnamela, ho un paio di idee che possiamo discutere  in ufficio sett prossima

**Comment by bacca87** [comment]
si ok ma ci sarebbero altri 26 issue da chiudere prima di questa che è abbastanza inutile se ancora siamo pieni di bug sulle funzionalità base...
